spec/features/item/new_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe 'As a visitor' do
+  describe "When I link to '/merchants/:merchant_id/items/new'" do
+    it "I see a form to add a new item" do
+      m1 = Merchant.create!(name: 'bobs burgers', address: '123 burger lane', city: 'burger city', state: 'burgerana', zipcode: 12345)
+      burger = m1.items.create!(name: "burger", description: "juicy and delish", price: 5.00, image: "https://media.gettyimages.com/photos/burger-for-4th-of-july-picture-id683734168?s=2048x2048", active: true, inventory: 50)
```

Comment:
Because you are testing for creating a new item, I would remove this line. You may run into a scenario where the information you're seeing on the page is from the old item rather the one you're trying to create